CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## master (unreleased)
 
+* [#11](https://github.com/rubocop-hq/rubocop-minitest/pull/11): Add new `Minitest/RefuteNil` cop. ([@tejasbubane ][])
```

Comment:
Can you add `### New features` category?

Reply:
added